Tests/UnitTests/Utilities/SecureStoreService+OneLoginTests.swift

```diff
@@ -0,0 +1,49 @@
+@testable import OneLogin
+import SecureStore
+import Testing
+
+struct SecureStoreServiceTests: ~Copyable {
+    var sut: SecureStoreService!
+    
+    init() throws {
+        let config = SecureStorageConfiguration(
+            id: "testConfig",
+            accessControlLevel: .open
+        )
+        sut = SecureStoreService(configuration: config)
+        
+        try sut.saveItem(
+            item: "testRefreshTokenExpiry",
+            itemName: OLString.refreshTokenExpiry
+        )
+        try sut.saveItem(
+            item: "testPersistentSessionID",
+            itemName: OLString.persistentSessionID
+        )
+        try sut.saveItem(
+            item: "testStoredTokens",
+            itemName: OLString.storedTokens
+        )
+    }
+    
+    deinit {
+        try? sut.delete()
+    }
+
+    @Test
```

Comment:
Should we explain this test a little better?

I suspect when doing this we may discover it may actually be testing a few different things - maybe each should be a different test?

```suggestion
    @Test("SecureStoreService throws an error when items are not found")
```

```suggestion
    @Test("Clear session data deletes the refresh token, persistentSessionID and tokens")
```